.tool-versions

```diff
@@ -0,0 +1,2 @@
+nodejs 16.19.0
+yarn 1.22.17
```

Comment:
Do you really want to set this version of yarn?

Reply:
Good question. Yes, and I was surprised by this as well. Explanation here: https://github.com/twuni/asdf-yarn#looking-for-yarn-v2

Reply:
Approving, though I feel like this is confusing. It's probably worth adding a note to the README for folks using ASDF with that link.

Reply:
I removed the yarn requirement/version. I misunderstood how asdf should be configured.